example/build.gradle.kts

```diff
@@ -42,7 +46,6 @@ repositories {
 java {
     toolchain {
         languageVersion.set(JavaLanguageVersion.of(17))
-        vendor.set(JvmVendorSpec.BELLSOFT)
```

Comment:
have no idea how to make it work with this specific vendor. I am using sdkman to get JVM.
Need your help figuring out how to make it work if you still need this line